toolkit/resources/imageconfigs/packagelists/accessibility-packages.json

```diff
@@ -0,0 +1,9 @@
+{
+    "packages": [
+        "espeak-ng",
+        "espeakup",
+        "kernel-drivers-sound",
```

Comment:
Nit: can we switch places of the drivers to maintain alphabetical order?:)

Reply:
Removed from this PR, but will address, thanks!